github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml

```diff
@@ -1606,6 +1606,44 @@ presubmits:
           privileged: true
       nodeSelector:
         type: bare-metal-external
+  - always_run: false
```

Comment:
Can you add `optional: true` ?

Reply:
sure, but aint it the default ?
i saw that we removed this all over and just using `always_run: false`